plugins/modules/events_and_notifications_workflow_manager.py

```diff
@@ -0,0 +1,2589 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2022, Cisco Systems
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+__author__ = ("Abhishek Maheshwari, Madhan Sankaranarayanan")
+
+DOCUMENTATION = r"""
+---
+module: events_and_notifications_workflow_manager
+short_description: Resource module for Network Device
+description:
+- Manage operations create, update and delete of the resource Network Device.
+- Adds the device with given credential.
+- Deletes the network device for the given Id.
+- Sync the devices provided as input.
+version_added: '6.8.0'
+extends_documentation_fragment:
+  - cisco.dnac.intent_params
```

Comment:
it is the  cisco.dnac.workflow_manager_params

Reply:
added.